app/src/main/java/fr/free/nrw/commons/upload/ThumbnailsAdapter.java

```diff
@@ -1,5 +1,7 @@
 package fr.free.nrw.commons.upload;
 
+import static com.mapbox.mapboxsdk.Mapbox.getApplicationContext;
```

Comment:
I don't think you are using correct context here

Reply:
Oh ! sorry for that, it was added by default and was working properly , so I didn't bother to change it. Now, I fixed that out.